src/XMakeTasks/UnitTests/DirectoryBuildProjectImportTestBase.cs

```diff
@@ -20,7 +20,7 @@ abstract public class DirectoryBuildProjectImportTestBase : IDisposable
                     </PropertyGroup>
                 </Project>";
 
-        private const string ProjectRelativePath = @"src\foo\foo.csproj";
+        private readonly string _projectRelativePath = Path.Combine("src", "foo", "foo.csproj");
```

Comment:
Yeah it went from a const to readonly because it's value is assigned at run time.  So technically the rename is necessary

---
In reply to: [76826798](https://github.com/Microsoft/msbuild/pull/974#discussion_r76826798) [](ancestors = 76826798)